R/test-threads.R

```diff
@@ -73,7 +73,7 @@ test_threads <- function(
   search_str <- "MAXEVAL|NITER"
```

Comment:
It won't let me comment on the actual line (because you didn't change it) but we should add `.overwrite = TRUE` to the `copy_model_from()` call.

You already have `overwrite = TRUE` in the `submit_models()` call (which I think is the right move), but if you try to call `test_threads()` and you forgot to cleanup before, it will fail. Calling `delete_mods()` is the obvious move, but this would likely happen in a new session when you don't have your model objects around anymore (and getting them back is actually kind of a pain, because the names are all annoying and constructed).